src/components/Tooltip/Tooltip.tsx

```diff
@@ -45,83 +88,8 @@ const TooltipBody = styled.div<Pick<TooltipProps, 'inverted'>>`
         border: 0.25rem solid rgba(0, 0, 0, 0);
         border-bottom-color: ${p => (p.inverted ? Colors.AUTHENTIC_BLUE_50 : Colors.AUTHENTIC_BLUE_900)};
         margin-left: -0.25rem;
```

Comment:
what is this negative left margin used for?

Reply:
The arrow